lib/db_helper/tasks.rb

```diff
@@ -208,23 +243,29 @@
 
   desc 'Remove sandbox'
   task :remove_sandbox do
-    default_sandboxes_base_dir = SandboxManager.default_sandboxes_base_dir
-    sandboxes_dir = InputReader.get_string(prompt: "Sandboxes path (#{default_sandboxes_base_dir}):", default_value: default_sandboxes_base_dir)
-    sm = SandboxManager.new(sandboxes_dir)
+    default_sandboxes_base_dir =
+      DbHelper::SandboxManager.default_sandboxes_base_dir
+    sandboxes_dir = InputReader.get_string(
+                      prompt: "Sandboxes path (#{default_sandboxes_base_dir}):",
+                      default_value: default_sandboxes_base_dir)
+    sm = DbHelper::SandboxManager.new(sandboxes_dir)
     ports = sm.ports
     selection_proc = ->(sandbox) { "#{sandbox.foreground(:cyan)}: #{ports[sandbox].foreground(:green)}" }
     sandboxes = ports.keys.sort_by { |sandbox| ports[sandbox] }
-    selected_sandbox = InputReader.select_item(sandboxes,
-                                               selection_attribute: selection_proc,
-                                               prompt: 'Which sandbox would you like to delete?')
+    selected_sandbox = InputReader.select_item(
+                         sandboxes,
+                         selection_attribute: selection_proc,
+                         prompt: 'Which sandbox would you like to delete?')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.